test/csuite/CMakeLists.txt

```diff
@@ -36,15 +36,17 @@ define_c_test(
     TARGET test_incr_backup
     SOURCES incr_backup/main.c
     DIR_NAME incr_backup
-    SMOKE
+    EXEC_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/incr_backup/smoke.sh
+    ARGUMENTS $<TARGET_FILE:test_incr_backup>
```

Comment:
Sorry @alisonfel, might be a stupid question but why do we sometimes specify the `WT_HOME` directory with the `-h` arg, and sometimes we don't?

Reply:
Not a stupid question at all! 
A majority of the tests are built with the `test_util` library, which supports passing the `-h` option to specify the DB home directory. Manually passing `-h` is mostly a defensive measure to ensure parallel executions of the tests don't step over each other and use the same home directory. Hence we are able pass the the home directory for the test definitions that don't use a wrapper script (since we are invoking the binary directly). 

For the test definitions that use an`EXEC_SCRIPT`, the majority of the wrapper scripts aren't really implemented to take in `-h` arguments, hence we don't end up passing them as arguments. This generally turns out ok since the script either sets a WT_HOME directory internally or it defaults to the working directory the script lives in. (Its more so an issue when we invoke the binary directly, where the working directory ends up being the root binary directory, where `ctest` is executed).

Reply:
Thanks for the explanation!